lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -133,12 +159,15 @@ def get_amplitude_course_recommendations(user_id, recommendation_id):
     return True, False, []
 
 
-def filter_recommended_courses(user, unfiltered_course_keys, recommendation_count=5):
+def filter_recommended_courses(
+    user, unfiltered_course_keys, recommendation_count=5, country_code=None, visiting_course=None
+):
     """
     Returns the filtered course recommendations. The unfiltered course keys
     pass through the following filters:
-        1. Remove courses that a user is already enrolled in
-        2. <To be added later>
+        1. Remove courses that a user is already enrolled in.
+        2. Remove the courses which is restricted in user region.
+        3. Remove course which user is currently visiting
```

Comment:
Keep the same order in which restrictions are applied.
```suggestion
        2. Remove course which user is currently visiting
        3. Remove the courses which is restricted in user region.
```